src/ety_main.erl

```diff
@@ -31,6 +31,10 @@ parse_args(Args) ->
              "Add path to the front of Erlang's code path"},
          {load_end,   $z,   "pz",        string,
              "Add path to the end of Erlang's code path"},
+         {force,       $f,   "force",      undefined,
```

Comment:
Is this the `--dirty` flag mentioned in #37 ?

Reply:
yes